CRM/Core/BAO/CustomField.php

```diff
@@ -1159,6 +1163,11 @@ public static function getDisplayValueCommon(
         $html_type == 'Multi-Select'
       )
     ) {
+      // Keep only the custom option 'key' => 'value' pair
+      // to avoid return of unnecessary data from the hook.
+      if (isset($option['attributes'])) {
+        unset($option['attributes']);
+      }
```

Comment:
I've removed the `$option` changes here. This was made to avoid any return of unnecessary data from the hook but I think it should be better handled in the hook itself. Thanks:)